### PR TITLE
update to js intro lesson

### PR DESCRIPTION
**Pull request by amaseda**
Converted this from a list-y explanation of Javascript basics to an exercise-based class.

**Questions**
- Should the Learning Objectives be condensed since we're not explicitly covering those points (unless brought up in the exercise check-ins)?
- [js-data-types](https://github.com/ga-wdi-exercises/js-data-types) is an assortment of small Javascript exercises. Is there enough variety? I don't expect them to finish this in class.
- Thoughts on current timing?

**Notes**
- Working on formatting truth tables in [js-data-types](https://github.com/ga-wdi-exercises/js-data-types).
- Need to format/add sections so that the `lesson-plan-checklist` test passes.
- The `Some Things To Keep In Mind` section is for basic Javascript takeaways that may not have been evident in the pre-work. These are sections I pulled out of the old lesson plan.

### CODE REVIEW

**Comment by RobertAKARobin** [comment]
Could you give the exercise its own repo in ga-wdi-exercises?

**Comment by amaseda** [comment]
@RobertAKARobin You mean this one? https://github.com/ga-wdi-exercises/js-data-types

**Comment by RobertAKARobin** [comment]
I'm an idiot.

**Comment by RobertAKARobin** [comment]
- Eh, I kinda like having the learning objectives there.
- I think it's great!
- Looks good to me!

Flying colors, all around.

**Comment by amaseda** [comment]
Word, thaaaaanks. Going to try to get another pair of 👀before merging.

File: readme.md [hunk 1]
```diff
@@ -1,292 +1,165 @@
...
+* Describe uses of mathematical operators in Javascript.
+* Define type coercion.
+* Define and use complex data types.
+* Explain the difference between `prompt` and `console.log`
+* Practice proper JS syntax and semantic variable naming.
+* Differentiate between true & false && truthy & falsey
+* Describe why control flow is utilized in computer programming
+* Write an if, else if, and else statement in JS
+* Write a for loop and while loop in JS and differentiate between them
+* Utilize loops to iterate through complex data types
```
Comment (jshawl) [inline readme.md:15]: these LO's look good to me :+1:

File: readme.md [hunk 2]
```diff
@@ -1,292 +1,165 @@
...
-  - What does it do?
-    1. (R)eads our code.
-    2. (E)valuates it.
-    3. (P)rints it to the console.
-    4. Then it (L)oops back to the beginning, ready to (R)ead the next line of code we feed it.
+### Think-Pair-Share: Identify JS Features in Cookie Clicker
 
-> `⌘ + ⌥ + i` enters you in the the chrome dev tools(if you're using chrome...) Here you can do a bunch of stuff like inspect elements and looks at the html. More importantly for this class though, is it allows you to access the console which interacts with the JS you loaded to your page. In our case we'll see that interaction with the code below
+**Think:** Spend three minutes playing around with Cookie Clicker and think about the following questions...
+* What Cookie Clicker features are powered by Javascript?
```
Comment (jshawl) [inline readme.md]: - What Cookie Clicker features are _not_ powered by Javascript?

File: readme.md [hunk 3]
```diff
@@ -1,292 +1,165 @@
...
+* Javascript: Behavior
 
-- Format: `var NAME = DATA;`
+## Javascript: The Client-Side Programming Language of the Web (5 minutes, 0:25)
 
-  ```javascript
-  // For example...
-  var myClass = "WDI7";
+**Brief history:** Created in 10 days by Brendan Eich, of Mozilla. *Not* related to Java in any way but its name.
+* "Java" is to "Javascript" as "ham" is to "hamster"
```
Comment (jshawl) [inline readme.md:66]: industry experience++

File: readme.md [hunk 4]
```diff
@@ -1,292 +1,165 @@
...
+### Why is it the dominant programming language of the web?
 
-  ```javascript
-  // This change is valid in Javascript.
-  var myFavoriteNumber = 5;
-  var myFavoriteNumber = "five";
-  ```
+Barriers to entry for learning Javascript are very low.
+* No additional software required to run it. Just a text editor and a browser.
+* You can even run it directly in the browser via its Javascript console.
```
Comment (jshawl) [inline readme.md]: `cmd-option-j`

File: readme.md [hunk 5]
```diff
@@ -1,292 +1,165 @@
...
+  * Ex. Hide images on the GA website.
+* On top of that, it's supported by all web browsers.
 
-## Numbers (10/45)
+Javascript has evolved since its creation.
+* One of the biggest additions to JS was AJAX, which allows use to reload parts of a page without refreshing the entire thing (just like on Facebook). Big implications for User Experience.
```
Comment (jshawl) [inline readme.md]: (Asynchronous Javascript and XML) and/or link to docs

File: readme.md [hunk 6]
```diff
@@ -1,292 +1,165 @@
...
+<!-- index.html -->
 
-  // Subtraction
-  10 - 2;
-  => 8
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Yo</title>
+    <script src="script.js"></script>
```
Comment (jshawl) [inline readme.md]: `type="text/javascript"`

File: readme.md [hunk 7]
```diff
@@ -1,292 +1,165 @@
...
+```
 
-  // Division
-  10 / 2;
-  => 5
-  ```
+## Next, open the site in Chrome and open the Dev Tools
 
-- Order of precedence: P(E)MDAS (Parentheses, Multiplication, Division, Addition, Subtraction)
+Open the Javascript console in the browser using the following shortcut: `Command + Option + I`
```
Comment (jshawl) [inline readme.md]: I is the element inspector, type esc key to toggle. Or cmd option j to go straight to console

**Comment by jshawl** [comment]
:+1: https://github.com/ga-wdi-exercises/js-data-types looks like a great exercise!

**Comment by jshawl** [comment]
Timing feels a little short. Maybe have a plan b in case things move too slowly or too quickly.

**Comment by amaseda** [comment]
@jshawl Thanks for all the comments. Implementing now then gonna merge.

**Pull request merged**